### PR TITLE
Bf cron jerb

### PR DESCRIPTION
**Pull request by billyfung**
- edited app.yaml
- created cron.yaml
- scrape_all to run scrapers

### CODE REVIEW

File: app.yaml [hunk 1]
```diff
@@ -10,6 +10,8 @@ threadsafe: yes
 
 # Handlers define how to route requests to your application.
 handlers:
+- url:/.*
+  script: scrape_all.py
```
Comment (jinpark) [inline app.yaml]: add `login: admin` so not everyone can hit it

https://cloud.google.com/appengine/docs/python/config/cron?hl=en

File: app.yaml [hunk 2]
```diff
@@ -10,6 +10,8 @@ threadsafe: yes
 
 # Handlers define how to route requests to your application.
 handlers:
+- url:/.*
```
Comment (jinpark) [inline app.yaml]: why not make this just `/cronjob`

**Comment by jinpark** [comment]
hmm does this actually work, I was under the impression that it needs to be handled by a web process, not just a python file

**Comment by billyfung** [comment]
IIAM

**Comment by jinpark** [comment]
-_-

File: app.yaml (hunk 2 above)
Comment (jinpark) [inline app.yaml]: actually why not `/scrape_all`

**Comment by billyfung** [comment]
I have no idea how to test this locally??

**Comment by jinpark** [comment]
go to the admin page,  click on cron on the left

File: cron.yaml [hunk 3]
```diff
@@ -0,0 +1,4 @@
+cron:
+- description: scrape coffee sites
+  url: /cronjobs
+  schedule: every monday 09:00
```
Comment (jinpark) [inline cron.yaml]: maybe every day or every other day?

**Comment by jinpark** [comment]
me and mr pleb talked about this. hes going to move it all the the flask app WHERE IT BELONGS /whip

**Comment by billyfung** [comment]
Yes senpai -.-

File: app.yaml [hunk 4]
```diff
@@ -9,7 +9,11 @@ api_version: 1
 threadsafe: yes
 
 # Handlers define how to route requests to your application.
+# changed to scrape_all.app to get rid of EventError: threadsafe cannot be enabled with CGI handler:
```
Comment (jinpark) [inline app.yaml]: this line is a lie

File: main.py [hunk 5]
```diff
@@ -1,6 +1,10 @@
 from flask import Flask, render_template
+import webapp2
```
Comment (jinpark) [inline main.py]: why webapp 2

File: main.py [hunk 6]
```diff
@@ -1,6 +1,10 @@
...
 from google.appengine.ext import ndb
 from models import Coffee
+from scrapers.stumptown import scrape_stumptown
```
Comment (jinpark) [inline main.py]: might be better to just import scrapers and just do `scrapers.stumptown.scrape_stumptown` and intelli

File: cron.yaml [hunk 7]
```diff
@@ -0,0 +1,4 @@
+cron:
+- description: scrape coffee sites
+  url: /cronjob
+  schedule: every 48 hours
```
Comment (jinpark) [inline cron.yaml:4]: new line but whatever its fine :D

**Comment by jinpark** [comment]
tested, works, no probs

good job mr pleb

**Pull request merged**